src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -210,6 +210,12 @@
   <data name="IDS_FeaturePatternMatching" xml:space="preserve">
     <value>pattern matching</value>
   </data>
+  <data name="IDS_FeatureExpressionBodiedAccessor" xml:space="preserve">
+    <value>expression body property accessor</value>
+  </data>
+  <data name="IDS_FeatureExpressionBodiedDeOrConstructor" xml:space="preserve">
```

Comment:
That's what it says. That is shorthand for Deconstructor Or Constructor.

---
In reply to: [77881977](https://github.com/dotnet/roslyn/pull/13543#discussion_r77881977) [](ancestors = 77881977)